securebanking-openbanking-uk-rs-simulator-server/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/api/obie/payment/v3_1_9/vrp/DomesticVrpsApiController.java

```diff
@@ -89,13 +88,15 @@ public ResponseEntity domesticVrpGet(
             String xReadRefundAccount,
             HttpServletRequest request,
             Principal principal
-    ) throws OBErrorResponseException {
+    ) {
+        log.debug("Searching for VRP payment by id '{}'", domesticVRPId);
         Optional<FRDomesticVrpPaymentSubmission> optionalVrpPayment = paymentSubmissionRepository.findById(domesticVRPId);
+
         if (!optionalVrpPayment.isPresent()) {
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Domestic VRP payment '" + domesticVRPId + "' " +
                     "can't be found");
         }
-        log.debug("Found VRP payment '{}'", domesticVRPId);
+        log.debug("Found VRP payment '{}'", optionalVrpPayment);
```

Comment:
please change this line to show the `domestic vrp id` instead of the Optional object.